posthog/models/element/sql.py

```diff
@@ -1,7 +1,8 @@
 GET_ELEMENTS = """
 SELECT
-    elements_chain, count(1) as count, event as event_type
+    elements_chain, count(_sample_factor) as count, event as event_type
```

Comment:
I think having `_sample_factor` here doesn't change anything from having `1` and just confuses the reader?

Reply:
Ah, yeah, I was robotically copying the docs without understanding 🙈 

But `count(_sample_factor)` and `count()` have the same results 👍